frontend/src/main/web/lib/components/DropDown.jsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import Actions from '../actions/Actions';
 import {PureRenderMixin} from 'react/addons';
```

Comment:
Should be able to do one import line for both these: `import React, { PureRenderMixin } from 'react/addons'`

This is because `react/addons` is "React with addons" rather than just the addons.